src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/EUSettingsViewModel.swift

```diff
@@ -0,0 +1,94 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import Combine
+import UIKit
+
+class EUSettingsViewModel {
+
+	// MARK: - Model.
+
+	class CountryModel {
+		let country: Country
+
+		init(_ country: Country) {
+			self.country = country
+		}
+	}
+
+	// MARK: - Attributes.
+
+	let countryModels: [CountryModel]
+
+	// MARK: - Initializers.
+
+	init(countries availableCountries: [Country]) {
+		self.countryModels = availableCountries
+			.sorted { $0.localizedName <  $1.localizedName }
```

Comment:
```suggestion
			.sorted { $0.localizedName.localizedCompare($1.localizedName) == .orderedAscending }
```

Reply:
Using localizedCompare ensures that umlautes and similar language features are sorted correctly :)